frontend/device/input.lua

```diff
@@ -528,6 +528,12 @@ function Input:handleKeyBoardEv(ev)
         end
     end
 
+    -- F11 is used on Linux to toggle fullscreen
+    if self:isEvKeyPress(ev) and keycode == "VPlus" and jit.os == "Linux" then
```

Comment:
This should be something like self.device:isDesktop() I believe? It could be OSX, it could (more theoretically) be Windows, etc.

Reply:
IIRC there's already a key sequence that does the native fullscreen switch on mac. It shouldn't hurt to map it to F11 cause function keys on mac by default are used for mac stuff things. Anyways I think it is handy to have it available on the emulator too, which is not desktop.

So maybe

`if self:evKeyPress(ev) and keycode = "F11" and not self:device:isAlwaysFullscreen()`

? I don't know. I'm open to recommendations :)

Reply:
> IIRC there's already a key sequence that does the native fullscreen switch on mac

That's Alt+F11 in my Xfce (which is default); my KWin doesn't seem to have any such binding by default, but presumably all WMs offer it.

> So maybe

Yes, that makes sense to me.

Reply:
> That's Alt+F11 in my Xfce

On Gnome that's F11 but Alt+F11 also works for me.